velox/common/caching/AsyncDataCache.cpp

```diff
@@ -654,7 +654,7 @@ void AsyncDataCache::backoff(int32_t counter) {
   size_t seed = folly::hasher<uint16_t>()(++backoffCounter_);
   const auto usec = (seed & 0xfff) * (counter & 0x1f);
   VELOX_CACHE_LOG_EVERY_MS(INFO, 1'000)
-      << "Backoff in allocation contention for " << usec << " us.";
+      << "Backoff in allocation contention for " << succinctMicros(usec);
```

Comment:
I wonder what does "Backoff in allocation contention" mean? Would you explain a bit?